### DISCUSSION TITLE
There should be a way to block people that the system selects

### DISCUSSION THREAD

**Discussion by clinhart** [community/community#54232] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I got stuff about somebody appear in my feed just because I collaborated with them in the past.
But I don't want to see anything from them - just seeing their name and face triggers me - because the collaboration was very toxic.
I'd like to have a feature to completely block anything from that person, so it will never show up in my feed again.

**Comment by bobwhitelock** [comment]
Will blocking the user do this? It says it will remove them as a collaborator. https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account
